main.c

```diff
@@ -430,9 +430,36 @@ static bool parse_mode(const char *value, int *width, int *height,
 	return true;
 }
 
+
+static void fixup_disabled_head(struct randr_head *head)
+{
+	struct randr_mode *mode;
+
+	if (head->scale <= 0.0) {
+		head->scale = 1.0;
```

Comment:
Maybe we can just set scale to 1.0 when initializing the `randr_head` instead?

Reply:
yeah, I actually thought about that one when looking at the code and then forgot ;)